addons/hw_drivers/views/pos_display.html

```diff
@@ -49,7 +49,7 @@
                         $('#iot-devices').html(iot_devices_html);
                     })
                     .catch((error) => {
-                        $('.error-message').removeClass('d-none').text('Error: ' + error);
+                        console.log('Error: ', error);
```

Comment:
Moved errors to console because most are not relevant to users

Reply:
don't really like this idea :/ although the clients are not relevant to the user, it might be for us. How are they supposed to know that something went wrong if we don't give any feedback on the matter for them.
The same way tracebacks shows on odoo's side when something went wrong, we don't expect them to understand it but to report it to their support service

Reply:
The errors here were only JavaScript errors, which disappeared 90% of the time when refreshing the page. As discussed with @Ysoroko, we have removed them.

Reply:
any reasons to use .log instead of .error ?